### ISSUE TITLE
about val score

### ISSUE THREAD

**Issue by cikers** [MILVLG/mcan-vqa#16]
Thank you very much for sharing your work!
I have a question.
I train the model in train_mode='train' and train_split='train', other settings are not change.
but the best score is no more than 65% in the val dataset.
In your work, the score is 66~67% in the val dataset.
What should i do for my confusion?
Thanks again!

**Comment by MIL-VLG** [comment]
can you provide the log file `results/log/log_run_<VERSION>.txt` for your training?

**Comment by cikers** [comment]
[log_run_30.txt](https://github.com/MILVLG/mcan-vqa/files/3621126/log_run_30.txt)
I use resume=True to train the model, so the first val score is up to 60.21%.

**Comment by yuzcccc** [comment]
It seems that you have modified the learning rate and removed the `warm-up strategy` for learning rate, which are very important to the final performance. I suggest checkout the latest version and use the default hyper-param instead to rerun the script

**Comment by cikers** [comment]
ok, I will check the code again.
Thank you.

**Comment by yuzcccc** [comment]
Have you reproduced the results with the default hyper-params?
